Explorer/Assets/DCL/Rendering/GPUInstancing/GPUInstancingRenderFeature.cs

```diff
@@ -0,0 +1,58 @@
+﻿using ECS;
+using System;
+using UnityEngine;
+using UnityEngine.Rendering.Universal;
+
+namespace DCL.Rendering.GPUInstancing
+{
+    public partial class GPUInstancingRenderFeature : ScriptableRendererFeature
+    {
+        [SerializeField] private GPUInstancingRenderFeature_Settings m_Settings;
+
+        public GPUInstancingRenderFeature_Settings Settings => m_Settings;
+
+        private GPUInstancingService instancingService;
+        private GPUInstancingRenderPass instancingRenderPass;
+        public GPUInstancingRenderFeature()
+        {
+            m_Settings = new GPUInstancingRenderFeature_Settings();
+        }
+
+        public override void Create()
+        {
+            instancingRenderPass = new GPUInstancingRenderPass(instancingService)
+            {
+                renderPassEvent = RenderPassEvent.BeforeRenderingOpaques
+            };
+        }
+
+        public void Initialize(GPUInstancingService service, IRealmData realmData)
+        {
+            instancingService = service;
+            instancingRenderPass?.SetService(service, realmData);
+        }
+
+        // public override void SetupRenderPasses(ScriptableRenderer _renderer, in RenderingData _renderingData) { }
```

Comment:
What's this?

Reply:
It was just a reminder of possible API that we can use when moving our Instancing System into RenderFeature. Removed for now